README.md

```diff
@@ -29,6 +29,13 @@ from the project root. This will create a plugin zip file at
 `bazel-bin/ijwb/ijwb_bazel.zip`, which can be installed directly
 from the IDE.
 
+For CLion, replace all mentions of `ijwb` above to `clwb`. If CLion refuses
+to load the plugin because of version issues, specify the `ij_product`
+manually. All supported product versions may be found in
+`intellij_platform_sdk/build_defs.bzl`.
+
+Example: `bazel build //clwb:clwb_bazel_zip --define=ij_product=clion-2018.1`
```

Comment:
Should use `clion-latest` instead.

Reply:
See my updated wording (coming in a few minutes). Note, that I think there is a minor regression because `clion-latest` is hardcoded to `clion-2017.3`.

Would you like to have no reference where to get more info for different `ij_product` versions?